src/thank-you/download.njk

```diff
@@ -1,5 +1,7 @@
 ---
-title: Thank you for dowloading...
+title: Thank you! 
+subtitle: [Click here](https://26586079.fs1.hubspotusercontent-eu1.net/hubfs/26586079/ebook_%20The%20Ultimate%20Beginner%20Guide%20to%20a%20Professional%20Node-RED-V1.pdf) to start reading The Ultimate Beginner Guide to a Professional Node-RED.
+description: We've also sent a copy to your inbox. 
```

Comment:
For the URL, you need to use HTML tags. I wasn't expecting you to add an URL in the subtitle.
And for both, you need to add double quotation marks, because otherwise, the special characters that are part of the text are confused as code.

With this change it should work

```suggestion
subtitle: "<a href='https://26586079.fs1.hubspotusercontent-eu1.net/hubfs/26586079/ebook_%20The%20Ultimate%20Beginner%20Guide%20to%20a%20Professional%20Node-RED-V1.pdf'>Click here</a> to start reading The Ultimate Beginner Guide to a Professional Node-RED"
description: "We've also sent a copy to your inbox" 
```